emperor/support_files/js/multi-model.js

```diff
@@ -0,0 +1,103 @@
+define(['underscore'],
+function(_) {
+//    /**
+//     * @class MultiModel
+//     *
+//     * A model object containing metrics merged from multiple underlying
+//     * DecompositionModel objects that share the same coordinate space.
+//     *
+//     * The multiModel contains min and max range of all model objects in the
+//     * scene, allowing you to draw axes or normalize data for a parallel plot.
+//     */
+  function MultiModel(underlyingModels) {
```

Comment:
Yes, I had to move the union ranges code out of the sceneplotview3d object and into a model class so that I could use it from the view classes.  I hate the name multi-model though.